tests/integration/test_model.py

```diff
@@ -40,7 +40,16 @@ def training_df(c):
     df = timeseries(freq="1d").reset_index(drop=True)
     c.create_table("timeseries", df, persist=True)
 
-    return training_df
+    return None
+
+
+@pytest.fixture()
+def gpu_training_df(c):
+    cudf = pytest.importorskip("cudf")
+    df = timeseries(freq="1d")
+    df = df.map_partitions(cudf.from_pandas).reset_index(drop=True)
```

Comment:
Any reason we opt for `map_partitions` here over `dask_cudf.from_dask_dataframe`?

Reply:
No reason . Happy to switch it.